API/tasks/Avatar.js

```diff
@@ -0,0 +1,47 @@
+module.exports = function(web3) {
+
+const ipfs = require('../js-ipfs-api.js');
```

Comment:
This is all done in the /Store container, we just need to send the task to that comntainer

Reply:
@kingflurkel - can you do that ? TYS

Reply:
gonna need a little help / clarification